packages/desktop-client/src/components/accounts/Header.js

```diff
@@ -0,0 +1,927 @@
+              onCreateRule={onCreateRule}
+              onScheduleAction={onScheduleAction}
+            />
+          )}
+          <Button
+            bare
+            disabled={search !== '' || filters.length > 0}
+            style={{ padding: 6 }}
+            onClick={onToggleSplits}
+            title={
+              splitsExpanded.state.mode === 'collapse'
+                ? 'Collapse split transactions'
+                : 'Expand split transactions'
+            }
+          >
+            {splitsExpanded.state.mode === 'collapse' ? (
+              <ArrowsShrink3
+                style={{
+                  width: 14,
+                  height: 14,
+                  color: 'inherit',
+                }}
+              />
+            ) : (
+              <ArrowsExpand3
+                style={{
+                  width: 14,
+                  height: 14,
+                  color: 'inherit',
+                }}
+              />
+            )}
+          </Button>
+          {account ? (
+            <View>
+              <MenuButton onClick={() => setMenuOpen(true)} />
+
+              {menuOpen && (
+                <AccountMenu
+                  account={account}
+                  canSync={canSync}
+                  canShowBalances={canCalculateBalance()}
+                  showBalances={showBalances}
+                  showCleared={showCleared}
+                  onMenuSelect={item => {
+                    setMenuOpen(false);
+                    onMenuSelect(item);
+                  }}
+                  onReconcile={onReconcile}
+                  onClose={() => setMenuOpen(false)}
+                />
+              )}
+            </View>
+          ) : (
+            <View>
+              <MenuButton onClick={() => setMenuOpen(true)} />
+
+              {menuOpen && (
+                <CategoryMenu
+                  onMenuSelect={item => {
+                    setMenuOpen(false);
+                    onMenuSelect(item);
+                  }}
+                  onClose={() => setMenuOpen(false)}
+                />
+              )}
+            </View>
+          )}
+        </Stack>
+
+        {filters && filters.length > 0 && (
+          <FiltersStack
+            filters={filters}
+            conditionsOp={conditionsOp}
+            onUpdateFilter={onUpdateFilter}
+            onDeleteFilter={onDeleteFilter}
+            onClearFilters={onClearFilters}
+            onReloadSavedFilter={onReloadSavedFilter}
+            filterId={filterId}
+            filtersList={filtersList}
+            onCondOpChange={onCondOpChange}
+          />
+        )}
+      </View>
+      {reconcileAmount != null && (
+        <ReconcilingMessage
+          targetBalance={reconcileAmount}
+          balanceQuery={balanceQuery}
+          onDone={onDoneReconciling}
+          onCreateTransaction={onCreateReconciliationTransaction}
+        />
+      )}
+    </>
+  );
+}
+
+function ReconcilingMessage({
```

Comment:
IMO the two reconciliation components should be moved out into their own file since they’re a bit complicated